src/integrant/core.cljc

```diff
@@ -13,10 +13,16 @@
 (defrecord Ref    [key] RefLike (ref-key [_] key))
 (defrecord RefSet [key] RefLike (ref-key [_] key))
 
+(defn- namespaced-keyword? [key]
+  (and (keyword? key) (namespace key)))
```

Comment:
`qualified-keyword?`
https://github.com/clojure/clojure/blob/841fa60b41bc74367fb16ec65d025ea5bde7a617/src/clj/clojure/core.clj#L1634-L1637

Reply:
That was added in 1.9 I believe.

Reply:
Nice catch. Since Integrant relies on Clojure 1.9, let's replace `namespaced-keyword?` with `clojure.core/qualified-keyword?` instead.